packages/affiliates/examples/dapp-mining/README.md

```diff
@@ -0,0 +1,30 @@
+# Dapp Mining Examples
+
+This contains examples of how to tag transactions with both Web3 and Ethers libraries.
+
+## Background
+
+"Dapp Mining" is the name of a new program being run by Risk Labs. The goal is to incentivize third party developers
+to build Dapps (Decentralized Applications) on top of the UMA infrastructure. Through these Dapp interfaces
+we can measure the amount of synthetic tokens minted by "tagging" certain transaction on chain with a developers
+address or unique identifier. The incentives come in the form of UMA token airdrops each week to developers who bring in liquidity
+into UMA EMP (Expiring Multi Party) contracts. The exact amounts and values of UMA airdroped may vary based on
+many factors, but essentially the more liquidity provided through a Dapp, the higher the proportion of weekly
+rewards earned.
```

Comment:
Might be good to highlight the fact that this is also an example implementation for third party developer miners who want to launch their own dapp mining programs. 

Also would be valuable to note that these rewards are intended to be a piece of the $UMA received for Developer mining by each contract. Maybe something along the lines of:

```suggestion
rewards earned. Typically the size of Dapp mining rewards will be a percentage of the developer mining rewards received by the contract that the Dapp mints for.

This reference implementation is also intended to be used by developer miners to create their own Dapp mining programs. With the same tagging mechanism, developer miners could payout a portion of their rewards to third party application developers or affiliate marketers.
```

Reply:
yea good point. I dont know exactly how to organize all these docs but I will add this in the next commit. may also want to link to various articles/tweets for more info.